frontend/src/pages/accounts/settings.page.tsx

```diff
@@ -157,6 +164,43 @@ const Settings: NextPage = () => {
                     {labelCollectionWarning}
                   </div>
                 </div>
+                <div className={styles.field}>
+                  <h2 className={styles["field-heading"]}>
+                    {l10n.getString("setting-label-api-key")}
+                  </h2>
+                  <div
+                    className={`${styles["copy-api-key-input"]} ${styles["field-content"]}`}
+                  >
+                    {/* TODO: Convert the following element to <output> when "hidden" */}
+                    <div className={styles["copy-api-key-display"]}>
+                      {profile.api_token}
+                    </div>
+                    <span className={styles["copy-controls"]}>
+                      <span className={styles["copy-button-wrapper"]}>
+                        <button
+                          className={styles["copy-button"]}
+                          title={l10n.getString("settings-button-copy")}
+                          aria-label={l10n.getString("settings-button-copy")}
+                          onClick={copyApiKeyToClipboard}
```

Comment:
issue: This is making your test fail - we can probably use `MouseEvent<HTMLButtonElement>` from React

Reply:
Part of the trick here is to parse TypeScript's sometimes inscrutable error messages. In this case, when you hover over `onClick`'s red squiggly line, it says (among others):

```
Type '(e: MouseEvent) => void' is not assignable to type 'MouseEventHandler<HTMLButtonElement>'.
```

The former is the type of `copyApiKeyToClipboard`: it's a function with one parameter (`e` of type `MouseEvent`) that returns nothing (`void`). `onClick`, however, expects a `MouseEventHandler<HTMLButtonElement>`. You can tell TypeScript that it is of that type as follows:

```diff
-   const copyApiKeyToClipboard = (e: MouseEvent) => {
+   const copyApiKeyToClipboard: MouseEventHandler<HTMLButtonElement> = (e) => {
```

(The types are pretty similar, but this provides some extra info, such as that `e.currentTarget` is an `HTMLButtonElement`, and thus that you can access e.g. `e.currentTarget.form` inside the handler. Which, of course, is the case for an `onClick` handler of a `<button>`.)